.circleci/config.yml

```diff
@@ -25,6 +25,25 @@ jobs:
           when: always
           command: isort -c .
 
+  static-code-analysis-flake8:
+    docker:
+      - image: circleci/python:2.7
```

Comment:
Sadly I need a separate job with Python 2. This is of course interim solution, and hopefully would change in near future,